includes/Products.php

```diff
@@ -528,8 +528,17 @@ public static function update_google_product_category_id( \WC_Product $product,
 	 */
 	public static function get_product_gender( \WC_Product $product ) {
 
-		// TODO: implement
-		return '';
+		if ( $product->is_type( 'variation' ) ) {
+			$gender = get_post_meta( $product->get_parent_id(), self::GENDER_META_KEY, true );
```

Comment:
Need to get the product object and use its `get_meta()` method here, as direct post meta functions are essentially deprecated for products and orders.